### PR TITLE
Admin can view and cancel events by category and venue

### PR DESCRIPTION
**Pull request by erinnachen**

### CODE REVIEW

File: app/views/admin/venues/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            <p id="event-#{event.id}"><%= link_to "#{event.name}, #{event.venue.city}", event_path(event) %></p>
+            <p> <%= event.format_date %> | <%=event.format_time%></p>
+          </div>
+        </div>
+      </div>
+    <% end %>
+  </div>
+</div>
+
+<%= render partial: "shared/mailing_list" %>
```
Comment (damwhit) [inline app/views/admin/venues/show.html.erb]: Admin probably doesn't care to sign up for the mailing list.

**Comment by damwhit** [comment]
@erinnachen  one comment for ya.  Other than that it looks good to me!  @cheljoh what do you think?

File: app/views/admin/venues/show.html.erb (hunk 1 above)
Comment (erinnachen) [inline app/views/admin/venues/show.html.erb]: I want to be included in the sweet sweet mailing list....

**Comment by cheljoh** [comment]
Looking good!

**Pull request merged**